__tests__/app.test.js

```diff
@@ -39,3 +40,61 @@ describe("/api/categories", () => {
     });
   });
 });
+describe("/api/reviews", () => {
+  describe("GET /api/reviews/:review_id", () => {
+    it("status: 200, response with an array of review objects with its properties  ", () => {
+      const review_id = 7;
+      
+      return request(app)
+        .get(`/api/reviews/${review_id}`)
+        .expect(200)
+        .then((res) => {
+          expect(res.body.review).toBeInstanceOf(Object);
```

Comment:
52 - we don't need this.  Your expect on line 54/55 implies that it is an object